showcase/src/assets/translations/en.json

```diff
@@ -331,6 +331,43 @@
         "TITLE": "Toast notification"
       }
     },
+    "DEMO_RBAC": {
+      "DIRECTIVES": {
+        "AUTHORIZATION": {
+          "HIDE_ON_PERMISSION": {
+            "TITLE": "Hide On Permission directive",
+            "SWITCH_UNAUTHORIZED_ROLE": "Switch the unauthorized role:",
+            "ONLY_VISIBLE_TO": "It is only visible to an authorized user that doesn't have the '{{ role }}' role."
+          },
+          "SHOW_ON_PERMISSION": {
+            "TITLE": "Show On Permission directive",
+            "SWITCH_AUTHORIZED_ROLE": "Switch the authorized role:",
+            "ONLY_VISIBLE_TO": "It is only visible to an authorized user that has the '{{ role }}' role."
+          },
+          "CONTENT_PROTECTED": "This content is protected.",
+          "TITLE": "Authorization directives"
+        }
+      },
+      "SERVICES": {
+        "AUTHORIZATION": {
+          "FUNCTIONALITY": "Functionality",
+          "DESCRIPTION_METHODS": "The <code>StarkRBACAuthorizationService</code> provides the necessary methods to know whether the user has an specific role or set of roles and whether it is an anonymous user.",
+          "DESCRIPTION_TRANSITION_HOOKS": "It also adds a \"transition hook\" to the Router to protect certain routes form being accessed by unauthorized users. The transition hook will prevent the user from navigating to any route with specific permissions requirements defined on those routes. On top of that, the user can be redirected to another route if such redirection is defined as part of the route permissions configuration.",
```

Comment:
Typo: `"It also adds a \"transition hook\" to the Router to protect certain routes from being accessed by unauthorized users. The transition hook will prevent the user from navigating to any route with specific permissions requirements defined on those routes. On top of that, the user can be redirected to another route if such redirection is defined as part of the route permissions configuration."`